diff --git a/main.nf b/main.nf
index f396f55..b53c9ff 100755
--- a/main.nf
+++ b/main.nf
@@ -176,7 +176,7 @@ include { make_refpkg_wf } from './modules/refpackage' params (
 
 )
 
-include { epang_place_classify_wf } from './modules/epang_place_classify' params (
+include { pplacer_place_classify_wf} from './modules/pplacer_place_classify' params (
     output: params.output,
     cmalign_mxsize: params.cmalign_mxsize,
     // Placer
@@ -269,10 +269,11 @@ workflow {
     //
     // STEP 3. Place and Classify
     //
-    epang_place_classify_wf(
+    pplacer_place_classify_wf(
         dada2_wf.out.sv_fasta,
         make_refpkg_wf.out.refpkg_tgz,
-        dada2_wf.out.sv_long
+        dada2_wf.out.sv_weights,
+    dada2_wf.out.sv_map
     )
     
 }
